src/content/en/updates/2018/12/badging-api.md

```diff
@@ -142,6 +163,8 @@ discussion.
 * [Public explainer][explainer]
 * [Tracking bug][cr-bug]
 * [ChromeStatus.com entry][cr-status]
+* Request an [origin trial token][ot-request]
```

Comment:
Does this magically link to https://developers.chrome.com/origintrials/#/trials/active? (the trial is live)

Reply:
This will link to https://developers.chrome.com/origintrials/#/view_trial/1711367858400788481, the definition for `[ot-request]` is at the bottom of the doc.